### PR TITLE
update supported apps

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @karlitschek @jospoortvliet @nickvergessen @oparoz

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @mar1u5

File: admin_manual/installation/apps_supported.rst [hunk 1]
```diff
@@ -6,28 +6,37 @@ AGPL Apps
...
+* Calendar
 * Collaborative Tags
-* Comments 
+* Comments
+* Contacts
 * Encryption
 * External Sites
 * External Storage
-* Nextcloud WebDAV Endpoint (handles old and new webdav endpoints)
+* WebDAV Endpoint (handles old and new webdav endpoints)
```
Comment (nickvergessen) [inline admin_manual/installation/apps_supported.rst]: alphabet is a mess

Comment (MorrisJobke) [inline admin_manual/installation/apps_supported.rst]: damn -.-

**Comment by karlitschek** [comment]
👍

File: admin_manual/installation/apps_supported.rst [hunk 2]
```diff
@@ -6,28 +6,37 @@ AGPL Apps
...
 * Notifications
 * Object Storage (Swift)
+* Password Policy
 * Provisioning API
 * Template Editor (for notification emails)
+* Theming
 * Update Notifications
 * User External
 * User LDAP
+* User SAML
```
Comment (oparoz) [inline admin_manual/installation/apps_supported.rst]: I would use something like Shibboleth/SAML authentication which may be more meaningful to people looking at this list.

**Comment by oparoz** [comment]
What about Windows Network Drive storage and Sharepoint?

**Comment by MorrisJobke** [comment]
> What about Windows Network Drive storage and Sharepoint?

I haven't seen a repo here. So I would go for the "put it in the list once we have something" 😉

**Comment by MorrisJobke** [comment]
> Should a list of supported backends be added to "External Storage"?

This is documented in the files_external setup. I don't think we want this, because then we need to write this for all the different apps: Like "Files Access Control supports to deny a request based on URL, IP, Browser, ..." which is kind of overkill for this rough overview list.

**Pull request merged**